src/site/server/index.markdown

```diff
@@ -8,40 +8,168 @@ has-permalinks: false
 # {{ page.title }}
 
 Dart is a great fit for server applications.
-For example, Dart provides support for
-[futures](/docs/tutorials/futures) in the core
-[async](https://api.dartlang.org/apidocs/channels/stable/dartdoc-viewer/dart:asy
-nc)
-library, a robust [stream API](/docs/tutorials/streams), and
-[WebSockets](/samples/websockets/).
 
-### Where can I find more information about writing server code in Dart?
+* Runs nearly everywhere
+* Does nearly everything
+* Easy to develop and debug
+* Scales out
 
-Get started with [tutorials](/docs/tutorials), [articles](/articles/),
-and other documentation. Once you're coding, consult the
+The Dart VM runs anywhere you can get a Linux, Windows, or Mac prompt.
+It has been ported to i32, x64, ARM, ARM64, and MIPS.
+
+You can access files, UDP and TCP sockets, HTTP, Web sockets, and more with the
```

Comment:
Web sockets -> WebSockets
?
(A quick web search shows mostly WebSocket, and some Web Socket)